apps/src/dropletUtils.js

```diff
@@ -158,52 +158,44 @@ standardConfig.categories = {
  * @returns {Array}
```

Comment:
never mind the first part, I see that codeFunctions is well documented below. It might still be helpful to know what dropletConfig.blocks is